performance/config.py

```diff
@@ -16,6 +16,9 @@
 class Config:
     ENV = 'prod'
 
+    REPORTS_S3_BUCKET = 'govukverify-hub-prod-billing-reports'
+    BILLING_DIRECTORY = 'billing'
```

Comment:
Ach... *this* variable is the one you aren't using :)

Reply:
:)  Yes! I put it in for posterity more than anything. But I can remove it

Reply:
Shrug, if properly named it's probably fine ackshly :)